SAMtags.tex

```diff
@@ -213,6 +214,9 @@ \subsection{Additional Template and Mapping data}
 \item[TC:i:\tagvalue{}]
 The number of segments in the template.
 
+\item[TS:A:\tagvalue{strand}]
+Strand (`{\tt +}' or `{\tt -}') of the transcript that the query sequence represents.
```

Comment:
Does a query sequence "represent" a transcript? If so, perhaps add another sentence to explain what this means to you.